    /**
     * @test BadDataException
     */

    public function testBadDataException()
    {
        // Given
        $e = new Exception\BadDataException('message');

        // Then
        $this->assertInstanceOf(Exception\MathException::class, $e);
    }
